### ISSUE TITLE
Interface cannot be specified instead of IP

### ISSUE THREAD

**Issue by t0mmyt** [locp/cassandra#152]
I would like to specify an interface to bind to instead of an IP

(I may not know or care what an individual nodes IP is in advance beyond the seed node(s)).

PR with you shortly

**Comment by t0mmyt** [comment]
https://github.com/locp/cassandra/pull/153

**Comment by dallinb** [comment]
Hi,

I guess there is no reason to not add this in.  However, Puppet itself will know the IP address when applying the catalogue, therefore this piece of code will work:

```puppet
class { 'cassandra':
    cluster_name    => 'MyCassandraCluster',
    endpoint_snitch => 'GossipingPropertyFileSnitch',
    listen_address  => "${::ipaddress}",
    num_tokens      => 256,
    seeds           => '110.82.155.0,110.82.156.3',
    auto_bootstrap  => false
  }
}
```

But then as I said, no reason not to put it in and possibly some would prefer to specify an interface.

**Comment by dallinb** [comment]
This was duplicated/fixed in #153 .
